src/providers/ContentShareProvider/docs/ContentShareProvider.mdx

```diff
@@ -22,8 +36,17 @@ The `ContentShareProvider` provides state and functionality for content sharing.
   // Whether or not the local user's content share is loading
   isLocalShareLoading: boolean;
 
-  // The chime attendee ID of the user sharing
-  sharingAttendeeId: string | null;
+  // Whether the user can start a content share based on current limits and `isLocalUserSharing`
+  canStartContentShare: boolean;
```

Comment:
We might want to change this to `hasReachedContentShareLimit` so it's decoupled from `isLocalUserSharing`, let me know your thoughts

Reply:
decouple +1

Reply:
if there an use case that builder can implement to kick off the 1st sharing when the 3rd starts? In this case, even though it reaches the max limit, but it literally is allowed to start content share.

Reply:
Today we don't allow three content shares from media side. So they cannot start a 3rd one, but they can stop  one content share first, then `canStartContentShare` becomes true and then start the 3rd (which is in fact the second content share stream in the meeting)

Reply:
Cool, I'll go with option 1 now. We can always add `hasReachedContentShareLimit` if we find it needed later.